lib/service/moduleGenerator.js

```diff
@@ -86,7 +86,7 @@ function generateModules(task) {
   const requestMetadata = {
     authenticatedUsername: task.request.username,
     authenticatedUserId: task.request.userId,
-    apiVersion: task.response.headers['x-kinvey-api-version'] || '3',
+    apiVersion: '5',
```

Comment:
I'd put this [here](https://github.com/Kinvey/flex-sdk/blob/master/lib/service/modules/baseStore.js#L14) and use `this._apiVersion` [here](https://github.com/Kinvey/flex-sdk/blob/master/lib/service/modules/userStore.js#L127). Just seems... tidier this way, but not necessary, I guess.